exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -162,13 +164,16 @@ pub fn panic_if_java_options() {
 pub struct JavaServiceFactory;
 
 impl ServiceFactory for JavaServiceFactory {
-    fn command(&mut self, command: &str) -> Option<Box<CommandExtension>> {
-        use exonum::helpers::fabric;
+    fn service_name(&self) -> &str {
+        "JAVA_SERVICE_FACTORY"
```

Comment:
I don't think we should fix it right now. It is a blocker for multiple Java services, but it's not an issue until we have actual support for them.